agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -1,113 +1,117 @@
-#script to log and readout pfeiffer TPG 366 gauge contoller
-#via Ethernet connection
-#Zhilei Xu, Tanay Bhandarkar
+# Script to log and readout pfeiffer TPG 366 gauge contoller
+# via Ethernet connection
+# Zhilei Xu, Tanay Bhandarkar
 
 import socket
 import numpy as np
-from ocs import ocs_agent, site_config, client_t
+from ocs import ocs_agent, site_config
 from ocs.ocs_twisted import TimeoutLock
 import time
 
 BUFF_SIZE = 128
 ENQ = '\x05'
-IP_ADDRESS = '10.10.10.20'
+
 
 class pfeiffer:
-    """
-    CLASS to control and retrieve data from the pfeiffer tpg366 
+    """CLASS to control and retrieve data from the pfeiffer tpg366
     pressure gauage controller
-    ip_address: ip address of the deivce
-    port: 8000 (fixed for the device)
-    
+
+
+    Args:
+        ip_address: IP address of the deivce
+        porti (int): 8000 (fixed for the device)
+
     Attributes:
        read_pressure reads the pressure from one channel (given as an argument)
        read_pressure_all reads pressrue from the six channels
        close closes the socket
-    
+
     Todo:
         -Generalize IP address inputs (input IP  via command line?)
     """
-    def __init__(self, ip_address=IP_ADDRESS, port=8000, timeout=10, f_sample = 2.5):
+    def __init__(self, ip_address, port, timeout=10,
+                 f_sample=2.5):
         self.ip_address = ip_address
         self.port = port
         self.comm = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self.comm.connect((self.ip_address, self.port))
         self.comm.settimeout(timeout)
-    
+
     def read_pressure(self, ch_no):
         """
         Function to measure the pressure of one given channel
         ch_no is the chanel to be measured (e.g. 1-6)
         returns the measured pressure as a float
-        
+
         Args:
             ch_no: The channel to be measured (1-6)
+
         Returns:
             pressure as a float
         """
-        msg = 'PR%d\r\n'%ch_no
+        msg = 'PR%d\r\n' % ch_no
         self.comm.send(msg.encode())
-        status = self.comm.recv(BUFF_SIZE).decode()#Could probably use this to catch exemptions
+        # Can use this to catch exemptions, for troubleshooting
+        status = self.comm.recv(BUFF_SIZE).decode()
```

Comment:
I just have a question about this. It looks like we're sending a command and reading the result to sort of flush out some sort of internal buffer. I think that because we're not doing anything with the result of that first command. Is this required, or still just a debugging statement?

Reply:
Mainly debugging. It's not necessary, I just didn't want to lose that snippet of code.